doc/CHANGELOG.rst

```diff
@@ -5,6 +5,9 @@ This changelog *only* contains changes from the *first* pypi release (0.5.4.3) o
 
 Latest Changes:
 - **1.1.2_dev0 - 2020-10-21**
+
+  - Add importlib.util to fix instability in Python 3.9
```

Comment:
Is it specific to 3.9? i was under the impression that it's a more general Python issue.

Reply:
Unfortunately I cant replicate it on 3.5 to 3.8 which are regularly tested. It was reported for 3.9 and one patch release of 3.7.  I suspect it is actually a site issue just starting python and checking sys.modules finds util with a working find_spec on all the versions i have available.  Some configuration must be out there which doesnt load it.  

If you have a recommended log entry that better describes the problem, just post it and I will make an update.  Unfortunately as I cant replicate it I cant determine what is trigger it.

Reply:
I updated the comment to be less specific about the version and more about the problem.   I also added comments in the code on the kludge line to make it clear the purpose.

Thanks for submitting a review of the PR!